ci/cpu/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash -ex
+
+export PATH=/conda/bin:/usr/local/cuda/bin:$PATH:$WORKSPACE
+export HOME=$WORKSPACE
+
+BRANCH="$(git rev-parse --abbrev-ref HEAD)"
+if [[ "$BRANCH" != "main" ]]; then
+  echo 'Branch release not main, exiting...';
+  exit 1;
+fi
+
+cat <<EOF > credentials.sh
+#!/bin/bash
+echo username=$GIT_USERNAME
+echo password=$GIT_PASSWORD
+EOF
+
+git config --global user.email "gputester@rapids.ai"
+git config --global user.name "GPU Tester"
+git config --global credential.helper "/bin/bash $WORKSPACE/credentials.sh"
```

Comment:
I'm unfamiliar with this command and can't seem to find any examples of this type of usage online. Do you have any resources you can share for learning more about this method?

Reply:
This work had already been done prior, but you can use custom helpers(incl. shell scripts) to set git credentials https://git-scm.com/docs/gitcredentials#_custom_helpers